config-bootstrapper/src/main/java/org/hypertrace/entity/type/client/EntityTypeServiceClient.java

```diff
@@ -0,0 +1,59 @@
+package org.hypertrace.entity.type.client;
+
+import io.grpc.Channel;
+import java.util.List;
+import java.util.concurrent.Callable;
+import java.util.stream.Collectors;
+import org.hypertrace.core.grpcutils.client.GrpcClientRequestContextUtil;
+import org.hypertrace.core.grpcutils.client.RequestContextClientCallCredsProviderFactory;
+import org.hypertrace.entity.type.service.v2.DeleteEntityTypesRequest;
+import org.hypertrace.entity.type.service.v2.EntityType;
+import org.hypertrace.entity.type.service.v2.EntityTypeServiceGrpc;
+import org.hypertrace.entity.type.service.v2.EntityTypeServiceGrpc.EntityTypeServiceBlockingStub;
+import org.hypertrace.entity.type.service.v2.QueryEntityTypesRequest;
+import org.hypertrace.entity.type.service.v2.UpsertEntityTypeRequest;
+import org.hypertrace.entity.type.service.v2.UpsertEntityTypeResponse;
+
+public class EntityTypeServiceClient {
+
+  private final EntityTypeServiceBlockingStub blockingStub;
+
+  public EntityTypeServiceClient(Channel channel) {
+    blockingStub = EntityTypeServiceGrpc.newBlockingStub(channel).withCallCredentials(
+        RequestContextClientCallCredsProviderFactory.getClientCallCredsProvider().get());
+  }
+
+  private <V> V execute(String tenantId, Callable<V> c) {
+    return GrpcClientRequestContextUtil.executeInTenantContext(tenantId, c);
+  }
+
+  public UpsertEntityTypeResponse upsertEntityType(String tenantId, EntityType entityType) {
```

Comment:
How many of these are used here? Also, please update package.

Reply:
All of these are used. Some in integration tests only though.
I think this package is appropriate. Are you recommending to put version?